lms/static/js/search/course/main.js

```diff
@@ -0,0 +1,21 @@
+RequireJS.require([
+    'jquery',
+    'backbone',
+    'js/search/course/search_app',
+    'js/search/base/routers/search_router',
+    'js/search/course/views/search_form',
+    'js/search/base/collections/search_collection',
+    'js/search/course/views/search_results_view'
+], function ($, Backbone, SearchApp, SearchRouter, CourseSearchForm, SearchCollection, CourseSearchResultsView) {
+
+    var courseId = $('#courseware-search-results').attr('data-course-id');
```

Comment:
But $.data() uses data from internal cache, while $.attr() unnecessarily goes through DOM tree making it slower in the end.